styleguide.css

```diff
@@ -0,0 +1,273 @@
+/* Normalize */
+
+html {
+	font-family: sans-serif;
```

Comment:
You might want to default to [system-ui fonts](https://github.com/jonathantneal/system-font-css/blob/gh-pages/system-font.css)